packer/install-dependencies.sh

```diff
@@ -33,6 +33,88 @@ installContainerd
 
 installImg
 
+DASHBOARD_VERSIONS="1.8.3 1.6.3"
+for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}; do
+    pullImg "k8s.gcr.io/kubernetes-dashboard-amd64:v${DASHBOARD_VERSION}"
+done
+
+EXECHEALTHZ_VERSIONS="1.2"
+for EXECHEALTHZ_VERSION in ${EXECHEALTHZ_VERSIONS}; do
+    pullImg "k8s.gcr.io/exechealthz-amd64:${EXECHEALTHZ_VERSION}"
+done
+
+ADDON_RESIZER_VERSIONS="1.8.1 1.7"
```

Comment:
why is one with a patch version and the other without? Is that a thing?

Reply:
cf. https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/k8s_versions.go#L15
cf. https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/k8s_versions.go#L142

Reply:
oh weird